Makefile

```diff
@@ -0,0 +1,62 @@
+this_makefile := $(lastword $(MAKEFILE_LIST)) # Used to automatically list targets
+.DEFAULT_GOAL := list # If someone runs "make", run "make list"
+
+# Source files to format, lint, and type check.
+LOCATIONS=src tests
+
+# Unless overridden, build conda environment using the package name.
+PACKAGE_NAME = pseudopeople
+SAFE_NAME = $(shell python -c "from pkg_resources import safe_name; print(safe_name(\"$(PACKAGE_NAME)\"))")
+
+about_file   = $(shell find src -name __about__.py)
+version_line = $(shell grep "__version__ = " ${about_file})
+PACKAGE_VERSION = $(shell echo ${version_line} | cut -d "=" -f 2 | xargs)
+PACKAGE_VERSION = $(shell echo $(shell pip list | grep -e "$(SAFE_NAME)") | cut -d " " -f2)
+
+
+# If CONDA_ENV_PATH is set (from a Jenkins build), use the -p flag when making Conda env in
```

Comment:
nit: move this comment down to right above where it's relevant (`CONDA_ENV_CREATION_FLAG = ...`)